lib/public/views/lhcPeriods/ActiveColumns/lhcPeriodsActiveColumns.js

```diff
@@ -22,6 +23,10 @@ export const lhcPeriodsActiveColumns = {
     name: {
         name: 'Name',
         visible: true,
+        filter: ({ namesFilterModel }) => textFilter(
+            namesFilterModel,
+            { id: 'namesFilterInput', class: 'w-75 mt1', placeholder: 'e.g. LHC22a, lhc23b, ...' },
```

Comment:
Try to avoid relying on ids for tests on filters (and for tests in general)
Prefer using the filter's class (see how filter is displayed) and children input tag

Reply:
removed